app/controllers/responsiblepeople/TrainingController.scala

```diff
@@ -72,18 +78,12 @@ trait TrainingController extends RepeatingSection with BaseController {
         (edit, cacheMap.getEntry[BusinessMatching](BusinessMatching.key)) match {
           case (true, _) => Redirect(routes.DetailedAnswersController.get(index, flow))
           case (false, Some(BusinessMatching(_, Some(BusinessActivities(acts, _, _, _)),_,_,_,_, _, _, _)))
-            if acts.exists(act => act == MoneyServiceBusiness || act == TrustAndCompanyServices)
+            if acts.exists(act => act == MoneyServiceBusiness || act == TrustAndCompanyServices) || appConfig.phase2ChangesToggle
```

Comment:
Move appConfig toggle call to start of line for readability?

Reply:
Done